src/store/index.ts

```diff
@@ -1,17 +1,17 @@
 import Vue from 'vue'
 import Vuex from 'vuex'
-import * as getters from './getters'
-import { EnvConfigI } from '@/plugins/getConfig'
+import getters from './getters'
 
-Vue.use(Vuex)
+/**
+ * Configures and returns Vuex Store.
+ */
+export function getVuexStore () {
+  Vue.use(Vuex)
 
-export interface RootStateI {
-  config?: EnvConfigI
-}
+  const store = new Vuex.Store({
+    state: { config: {} },
```

Comment:
This line probably does nothing.

Eventually I expect this file to look a lot like the ones in our other apps.